src/python/TaskWorker/Actions/PostJob.py

```diff
@@ -1986,6 +2000,27 @@ def execute_internal(self):
                 return self.check_retry_count(80001), retmsg
             self.logger.info("====== Finished upload of output files metadata.")
 
+            if ASO_JOB:
+                self.logger.info("====== About to update publication flags of trasnfered files.")
+                for doc in getattr(ASO_JOB, 'docs_in_transfer', []):
+                    doc_id = doc.get('doc_id')
+                    self.logger.debug("Found doc %s" % doc_id)
+                    if doc.get('delayed_publicationflag_update'):
+                        newDoc = {
+                            'subresource' : 'updatePublication',
+                            'list_of_ids' : doc_id,
+                            'list_of_publication_state' : 'NEW',
+                            'asoworker' : '%'
+                        }
+                        try:
+                            self.server.post(self.rest_uri_no_api + "/filetransfers", data=encodeRequest(newDoc))
+                        except Exception as ex:
+                            retmsg = "Fatal error uploading output files metadata: %s" % (str(ex))
```

Comment:
Minor thing but this should probably be a different failure message ("Fatal error updating publication flags for transferred files" or "Fatal error requesting publication for transferred files" or something). Also, you left some typos in the word 'transferred' (trasnfered), lines 2004, 2020, 2022 :)